.env.mainnet

```diff
@@ -1,6 +1,6 @@
 VUE_APP_WS_URL=wss://backend.explorer.rsk.co
+VUE_APP_APPS_URL=""
 VUE_APP_STATS_URL=https://stats.rsk.co
-VUE_APP_GA_TAG=''
-VUE_APP_HOTJAR_ID=''
+VUE_APP_GA_TAG=GTM-56J7JTD
+VUE_APP_HOTJAR_ID=1857874
```

Comment:
Is OK to push these values?

Reply:
Yes, they're loaded by the client on page visit.